models/image/src/image-draft/transformers.ts

```diff
@@ -0,0 +1,20 @@
+import { Transformer } from '@commercetools-test-data/core';
+import type { TImageDraft, TImageDraftGraphql } from '../types';
+
+const transformers = {
+  graphql: Transformer<TImageDraft, TImageDraftGraphql>('graphql', {
+    buildFields: [],
+    replaceFields: ({ fields }) => {
+      return {
+        ...fields,
+        dimensions: {
+          ...fields.dimensions,
+          __typename: 'DimensionsInput',
+        },
+        __typename: 'ImageInput',
+      };
+    },
+  }),
+};
```

Comment:
I _love_ that we only have a graphql transform for this. Really lets consumers know that _this is the one use case_